### PR TITLE
Configure and create helpers

### PR DESCRIPTION
**Pull request by PetrosBompotis**
- Change include_all_helpers config to false
- Split the _collapsible_elements.html.erb file's content into partials and extract logic from the file into partials

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -23,5 +23,6 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+    config.action_controller.include_all_helpers = false
```
Comment (anthonyrouss) [inline config/application.rb:26]: Why did you add this configuration?

Comment (PetrosBompotis) [reply config/application.rb:26]: Normally, Rails loads all helper files in all views, which I don't prefer because it might lead to method names conflicting between different helpers. So, I changed a setting to stop this default behavior but i was wrong. Let me try another way with the addition of the helper inside the controller.
